libraries/AP_RangeFinder/examples/RFIND_test/RFIND_test.cpp

```diff
@@ -19,7 +19,7 @@ void setup()
     hal.console->printf("Range Finder library test\n");
 
     // setup for analog pin 13
-    AP_Param::set_object_value(&sonar, sonar.var_info, "_TYPE", RangeFinder::RangeFinder_TYPE_PLI2C);
+    AP_Param::set_object_value(&sonar, sonar.var_info, "_TYPE", RangeFinder::RangeFinder_TYPE_PLI2CV3HP);
```

Comment:
I think we can either drop the example change or use define to selection which rangefinder to use

Reply:
I have dropped the example change, we can edit the lidar type directly as we want.